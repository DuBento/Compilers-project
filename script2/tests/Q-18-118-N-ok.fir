int *factorial(int n) {
  if n > 1 then factorial = n * factorial(n-1); else factorial = 1;
}

int *fir() -> 0
@ { int f = 5; }
{
  writeln 'Teste para a função factorial';
  writeln f, ' != ', factorial(f);
}
>> { writeln 'End of function!'; }