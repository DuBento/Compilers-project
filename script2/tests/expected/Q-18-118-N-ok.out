Teste para a função factorial
5 != 120
End of function!
